src/Core/src/MauiContextExtensions.cs

```diff
@@ -67,5 +68,21 @@ public static void InitializeScopedServices(this IMauiContext scopedContext)
 			foreach (var service in scopedServices)
 				service.Initialize(scopedContext.Services);
 		}
+
+#if WINDOWS || ANDROID || IOS
+		public static FlowDirection GetFlowDirection(this IMauiContext mauiContext)
+		{
+			var appInfo = AppInfo.Current;
+
+			if (appInfo.RequestedLayoutDirection == LayoutDirection.RightToLeft)
+				return FlowDirection.RightToLeft;
+
+			return FlowDirection.LeftToRight;
+		}
+#else
+		public static FlowDirection GetFlowDirection(this IMauiContext mauiContext) =>
+			FlowDirection.LeftToRight;
+#endif
```

Comment:
Why this special case? Because Essentials throws? We can make Essentials return unknown. I notice that this is a far better experience for some apis. Like DeviceInfo for example. Some properties return unknown

I think we should actually do this since we are toting that testability concept. Right now a slew of exceptions is a bit overkill.

Reply:
Maybe make the netstd on AppInfo return Unspecified for theme and this? Maybe the same for the packaging model - might have to add an Unknown=0 to the enum.

No idea why I did not add it :(

Reply:
Yea I was mainly just trying to stick to the essentials pattern of throwing an exception from NS.

Changed it to just return `Unknown` and removed the `if/defs`